intro/notebooks-intro/notebooks-intro.ipynb

```diff
@@ -0,0 +1,311 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "1. Edit the following markdown cell with the *CHANGE_ME* text.\n",
+    "\n",
+    "Click the cell text to select the cell.\n",
+    "\n",
+    "Then, press `Enter` to edit the cell.\n",
+    "Alternatively, you can double click the cell.\n",
+    "\n",
+    "To render the cell, press `Ctrl+Enter`.\n",
+    "You can also render the cell and advance to the next cell by pressing `Shift+Enter`.\n",
+    "If no next cell exists, then pressing `Shift+Enter` adds a new cell at the bottom of the notebook.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
This cell should be a markdown cell
```suggestion
   "cell_type": "markdown",
```

Reply:
[a2c6e87](https://github.com/RedHatTraining/AI26X-apps/pull/4/commits/a2c6e8705eab488a26d3adc619ba98dcc5b6dab9)